src/main/java/dev/hse/cmc/process/ProcessingService.java

```diff
@@ -16,19 +15,8 @@ public class ProcessingService {
     private final KafkaProducerService kafkaProducerService;
 
     public void process(CryptoCurrencyData cryptoCurrencyData) {
-        CryptoCurrencyFiltered filtered = CryptoCurrencyFiltered.builder()
-            .symbol(cryptoCurrencyData.getSymbol())
-            .vendor(1L)
-            .observedAt(cryptoCurrencyData.getQuote().getUsd().getLast_updated().toInstant().atZone(
-                ZoneId.of("UTC+3")).toLocalDateTime())
-            .maxSupply((long) cryptoCurrencyData.getMax_supply())
-            .circulatingSupply((long) cryptoCurrencyData.getCirculating_supply())
-            .totalSupply((long) cryptoCurrencyData.getTotal_supply())
-            .volume24h(cryptoCurrencyData.getQuote().getUsd().getVolume_24h())
-            .marketCap(cryptoCurrencyData.getQuote().getUsd().getMarket_cap())
-            .priceUSD(cryptoCurrencyData.getQuote().usd.getPrice())
-            .build();
-
+        CryptoCurrencyFiltered filtered = FilterMapper.INSTANCE.creatingFilteredFromRawData(
```

Comment:
After refactoring `peace-death` of mapstruct you can leave it like that because it's looking pretty good